### ISSUE TITLE
phrase gets split on '

### ISSUE THREAD

**Issue by mwargan** [Donatello-za/rake-php-plus#14]
Same as except with newest version: https://github.com/Donatello-za/rake-php-plus/issues/2

When the text to be parsed includes a contraction the phrase returned is split on the apostrophe.

Example:
$text = "the rep doesn't really have anything to talk about"
returns
Array
(
[rep doesn] => 4
[talk] => 1
)

**Comment by Donatello-za** [comment]
Are you sure you are on the latest version (1.0.10.b-dev)? Look in your `vendor/donatello-za/rake-php-plus/composer.json` file. Also, what language are you using?

When I run it on the phrase `the rep doesn't really have anything to talk about` using `en_US` like this:
```php
$phrase_scores = RakePlus::create("the rep doesn't really have anything to talk about", 'en_US')->scores();
print_r($phrase_scores);
```
I get the following results:

```php
Array
(
    [rep] => 1
    [talk] => 1
)
```

**Comment by mwargan** [comment]
Hey! Yes (versions : * v1.0.10), I tried with the following:

> From one user a day to millions, your website can scale across servers so seamlessly that you won't even notice it happen. If there's a temporary surge in people visiting your website, we'll automatically increase server size to allow each of them to have an incredibly fast experience. When the surge is over, we'll scale back the server size to help you save costs.

and got the first result back as `'ll automatically increase server size` :/.

**Comment by Donatello-za** [comment]
Hi, this should be fixed in v1.0.11, sorry for the delay.
